_posts/2020-06-22-notes-from-community-call-3-june-2020.md

```diff
@@ -26,16 +26,16 @@ categories:
 
 ### Introductions
 
-Renato joined the call for the first time and was curious about the Standard, and had heard about the new 'open by default' policy in the Netherlands. He shared a couple of links he found inspiring: 
+Renato joined the call for the first time and was curious about the Standard, and had heard about the new 'open by default' policy in the Netherlands. He shared a couple of links he found inspiring:
 
-* [Creating new collaborative organisations to operate shared services](https://www.dxw.com/2020/02/creating-new-collaborative-organisations-to-operate-shared-services/), a blogpost by Richard Pope and DXW
+* [Creating new collaborative organizations to operate shared services](https://www.dxw.com/2020/02/creating-new-collaborative-organizations-to-operate-shared-services/), a blogpost by Richard Pope and DXW
```

Comment:
This breaks the link, the URL is spelled with a "s".

Reply:
Good spot; I was moving too quickly. I've fixed that now.